descope/management/jwt.py

```diff
@@ -0,0 +1,36 @@
+from descope.auth import Auth
+from descope.exceptions import ERROR_TYPE_INVALID_ARGUMENT, AuthException
+from descope.management.common import MgmtV1
+
+
+class JWT:
+    _auth: Auth
+
+    def __init__(self, auth: Auth):
+        self._auth = auth
+
+    def updateJWT(self, jwt: str, customClaims: dict) -> dict:
```

Comment:
in other places in the sdk we called the argument "token" and not jwt so maybe it worth to keep the same naming.. wdyt?

Reply:
done